spec/acceptance/organizer/add_to_context_spec.rb

```diff
@@ -14,6 +14,8 @@ def self.steps
         # with the value of 0, so it's available for
         # AddsOneAction
         add_to_context(:number => 0),
+        add_to_context(:message => "Just a message"),
+        add_to_context(:error_code => "D0H"),
```

Comment:
I wonder if we should just raise an error when someone tries to set these "reserved" keys via `add_to_context`. What do you think?

Reply:
I was thinking the exact same thing, and that I should raise an exception, but was concerned about breaking backwards compat.

That being said, if you're injecting a `message` context key using `add_to_context` you're probably doing so with full knowledge that it's not the smartest move.

Also, LS is pre1.0, so breaking changes are not a big deal. Lemme make those changes. Stand by.